colossalai/booster/plugin/torch_ddp_plugin.py

```diff
@@ -62,8 +62,12 @@ def save_sharded_model(self,
         if self.coordinator.is_master():
             super().save_sharded_model(model, checkpoint_path, gather_dtensor, prefix, max_shard_size, use_safetensors)
 
-    def save_sharded_optimier(self, optimizer: Optimizer, checkpoint: str, gather_dtensor: bool, prefix: str,
-                              size_per_shard: int):
+    def save_sharded_optimier(self,
+                              optimizer: Optimizer,
+                              checkpoint: str,
+                              gather_dtensor: bool = False,
```

Comment:
I find that the default value of `gather_dtensor` of difference methods are different. Please use the same value.